File: Modul 6. Array/PRAK603/PRAK603-2310817120003-NatalieGraceKatiandagho.c
#include <stdio.h>

int main()
{
    int n1;
    int n2;
    scanf("%d %d", &n1, &n2);

    if (n1 != n2)
    {
        printf("Jumlah tidak sama\n");
    }
    else
    {
        int x[n1];
        int y[n2];

        for (int i = 0; i < n1; i++)
        {
            scanf("%d", &x[i]);
        }

        for (int i = 0; i < n2; i++)
        {
            scanf("%d", &y[i]);
        }

        for (int i = 0; i < n1; i++)
        {
            printf("%d ", x[i] * y[i]);
        }
        printf("\n");
    }

    return 0;
}

File: Modul 6. Array/PRAK603/PRAK603-2310817120003-NatalieGraceKatiandagho.py
n1, n2 = map(int, input().split())

if n1 != n2:
    print("Jumlah tidak sama")
else:
    kali = [int(j) for j in input().split()]
    kali1 = [int(j) for j in input().split()]

    hasil = [k * k1 for k, k1 in zip(kali, kali1)]
    print(*hasil)
